[2025-10-11T07:43:07.966+0200] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: predict_batch_dag.send_batch_predictions scheduled__2025-10-11T05:22:35+00:00 [queued]>
[2025-10-11T07:43:07.968+0200] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: predict_batch_dag.send_batch_predictions scheduled__2025-10-11T05:22:35+00:00 [queued]>
[2025-10-11T07:43:07.968+0200] {taskinstance.py:2193} INFO - Starting attempt 3 of 1
[2025-10-11T07:43:07.973+0200] {taskinstance.py:2214} INFO - Executing <Task(_PythonDecoratedOperator): send_batch_predictions> on 2025-10-11 05:22:35+00:00
[2025-10-11T07:43:07.976+0200] {standard_task_runner.py:60} INFO - Started process 47425 to run task
[2025-10-11T07:43:07.979+0200] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'predict_batch_dag', 'send_batch_predictions', 'scheduled__2025-10-11T05:22:35+00:00', '--job-id', '244', '--raw', '--subdir', 'DAGS_FOLDER/prediction_dag.py', '--cfg-path', '/var/folders/rp/syznybx160jd0kg9k7r_sj8w0000gn/T/tmpi5ghkiz7']
[2025-10-11T07:43:07.980+0200] {standard_task_runner.py:88} INFO - Job 244: Subtask send_batch_predictions
[2025-10-11T07:43:07.999+0200] {task_command.py:423} INFO - Running <TaskInstance: predict_batch_dag.send_batch_predictions scheduled__2025-10-11T05:22:35+00:00 [running]> on host ebots-macbook-pro.local
[2025-10-11T07:43:08.018+0200] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='predict_batch_dag' AIRFLOW_CTX_TASK_ID='send_batch_predictions' AIRFLOW_CTX_EXECUTION_DATE='2025-10-11T05:22:35+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-11T05:22:35+00:00'
[2025-10-11T07:43:08.019+0200] {prediction_dag.py:41} INFO - ðŸ“„ Found CSV file: /Users/ebotfabien/Desktop/school/air_new/dags/destination/data_part_17.csv
[2025-10-11T07:43:08.026+0200] {prediction_dag.py:68} INFO - ðŸ“¦ Sending 10 batch(es)...
[2025-10-11T07:43:08.028+0200] {prediction_dag.py:73} INFO - ðŸ“¦ Sending batch 1/10
[2025-10-11T07:43:46.608+0200] {local_task_job_runner.py:121} ERROR - Received SIGTERM. Terminating subprocesses
[2025-10-11T07:43:46.615+0200] {process_utils.py:131} INFO - Sending Signals.SIGTERM to group 47425. PIDs of all processes in the group: [47425]
[2025-10-11T07:43:46.616+0200] {process_utils.py:86} INFO - Sending the signal Signals.SIGTERM to group 47425
[2025-10-11T07:43:50.356+0200] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: predict_batch_dag.send_batch_predictions scheduled__2025-10-11T05:22:35+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2025-10-11T07:43:50.356+0200] {taskinstance.py:1969} INFO - Dependencies not met for <TaskInstance: predict_batch_dag.send_batch_predictions scheduled__2025-10-11T05:22:35+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2025-10-11T07:43:50.358+0200] {local_task_job_runner.py:160} INFO - Task is not able to be run
